packages/sdk/src/swap/sdk.ts

```diff
@@ -238,10 +239,17 @@ export class SwapSDK {
   getStatus(
     swapStatusRequest: SwapStatusRequest,
     options: ApiService.RequestOptions = {},
-  ): Promise<SwapV2.SwapStatusResponse> {
+  ): Promise<SwapStatusResponse> {
     return ApiService.getStatus(this.options.backendUrl, swapStatusRequest, options);
   }
 
+  getStatusV2(
```

Comment:
could do something like this until we know the shape will stah the same. but no strong opinion
```suggestion
  /** @experimental */
  getStatusV2(
```